.github/workflows/frontend-collab-tests.yml

```diff
@@ -0,0 +1,54 @@
+# Leave the powered by Sauce Labs bit in as this means we get additional concurrency
+name: "Frontend Collaboration tests powered by Sauce Labs"
+
+on: [push]
+
+jobs:
+  withoutplugins:
+    name: without plugins
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Generate Sauce Labs strings
+      id: sauce_strings
+      run: |
+        printf %s\\n '::set-output name=name::${{ github.workflow }} - ${{ github.job }}'
+        printf %s\\n '::set-output name=tunnel_id::${{ github.run_id }}-${{ github.run_number }}-${{ github.job }}'
+
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 12
+
+    - name: Install all dependencies and symlink for ep_etherpad-lite
+      run: src/bin/installDeps.sh
+
+    - name: export GIT_HASH to env
+      id: environment
+      run: echo "::set-output name=sha_short::$(git rev-parse --short ${{ github.sha }})"
+
+    - name: Create settings.json
+      run: cp settings.json.template settings.json
+
+    - name: Disable import/export rate limiting
+      run: |
+        sed -e '/^ *"importExportRateLimiting":/,/^ *\}/ s/"max":.*/"max": 0/' -i settings.json
+
+    - uses: saucelabs/sauce-connect-action@v1
+      with:
+        username: ${{ secrets.SAUCE_USERNAME }}
+        accessKey: ${{ secrets.SAUCE_ACCESS_KEY }}
+        tunnelIdentifier: ${{ steps.sauce_strings.outputs.tunnel_id }}
```

Comment:
The PR description say 5 tunnels, but I only see one here. Where are the others?

Reply:
My bad.  5 browser clients.  That's just one tunnel and 5 VMs (or whatever sauce labs call them)

Reply:
Resolved